src/main/kotlin/io/github/anki/anki/controller/exceptions/DoesNotExist.kt

```diff
@@ -0,0 +1,6 @@
+package io.github.anki.anki.controller.exceptions
+
+interface DoesNotExist {
```

Comment:
Is this interface really necessary? Might it overhead?

Reply:
i wanted to create 1 method to handle that card or deck does not exist
so i made this interface and upcasted my exception to him in ControllerExceptionHandler